R/sql_translate_env.R

```diff
@@ -68,7 +68,13 @@ sql_translate_env.PrestoConnection <- function(con) {
       pmin = sql_prefix("LEAST"),
       is.finite = sql_prefix("IS_FINITE"),
       is.infinite = sql_prefix("IS_FINITE"),
-      is.nan = sql_prefix("IS_NAN")
+      is.nan = sql_prefix("IS_NAN"),
+      `[[` = function(x, i) {
+        if (is.numeric(i) && isTRUE(all.equal(i, as.integer(i)))) {
+          i <- as.integer(i)
+        }
+        dbplyr::build_sql(dbplyr::ident_q(x), "[", i, "]")
```

Comment:
Why do you need the `ident_q`?

Reply:
You're right, it's unnecessary given that `build_sql()` already protects us from malicious input.